drainer/translator/translator.go

```diff
@@ -74,7 +96,18 @@ func insertRowToDatums(table *model.TableInfo, row []byte) (pk types.Datum, datu
 			//
 			// Will get -1 here, note: uint64(int64(-1)) = 18446744073709551615
 			// so we change it to uint64 if the column type is unsigned
-			datums[col.ID] = fixType(pk, col)
+			datums[col.ID] = fixType(pk[0], col)
+		case clusteredPk:
+			for idxColOrdinal, idxCol := range commonPKInfo.Columns {
```

Comment:
will there be a case that `table.IsCommonHandle` is true but no PK? so `commonPKInfo` is nil

Reply:
no~ clustered index must be defined with a PK now, add a protection check at https://github.com/pingcap/tidb-binlog/pull/1040/files#diff-d362ef1721274a90486b677b50c57cfec86fac0f7fcf567587b452fe7611898dR61